### DISCUSSION TITLE
Shortcut button which opens up multiple websites at once

### DISCUSSION THREAD

**Discussion by 9662e103-129a** [dailydotdev/daily#898] [Feature Requests]
Was just thinking maybe there could be a shortcut button (just like the ones below) that can open up multiple websites at once: say I want to have it open up Twitter, Github, and some other documentation website. That would seem like a useful tool to increase efficiency!

![image](https://github.com/dailydotdev/daily/assets/92825997/7881ebdf-e2ca-4e23-9ccb-68d0185515ca)

**Comment by idoshamun** [comment]
Hi @win21H2,

That's an interesting suggestion but unfortunately out of our scope right now. I'm sure there are many other extensions that provide this capability. We're focusing on improving the core value of daily.dev for discovering new content and collaborating with other developers
